hack/builder/common.sh

```diff
@@ -0,0 +1,5 @@
+DOCKER_PREFIX=${DOCKER_PREFIX:-"quay.io/kubevirt"}
+DOCKER_IMAGE=${DOCKER_IMAGE:-"builder"}
```

Comment:
nit: Would it be better to go with something like `CONTAINER_PREFIX` rather than `DOCKER_PREFIX` ? 

Same would go for the image variable too.

Reply:
I thought about this but currently in KubeVirt repo we use DOCKER_PREFIX and DOCKER_IMAGE in other places. So I wanted to make it consistent. And if we are changing this we should consider changing this in other places as well.

Reply:
Yes that make sense to me - thanks.